Exec/Cases/PremBunsen2D/pelelm_prob.H

```diff
@@ -76,13 +75,12 @@ void pelelm_initdata(int i, int j, int k,
           for (int n = 0; n < NUM_SPECIES; n++){
             massfrac[n] = pmf_vals[3 + n];
           }
-          //eos.X2Y(molefrac, massfrac);
        }
     }
 
     AMREX_D_TERM(state(i,j,k,VELX) = 0.0;,
-                 state(i,j,k,VELX) = 0.1;,
-                 state(i,j,k,VELX) = 0.0);
+                 state(i,j,k,VELY) = 0.1;,
```

Comment:
should the 0.1 be on the X component still?

Reply:
It should be on the Y for this configuration. Not sure how it ended up on X ...